scripts/mount

```diff
@@ -259,6 +259,7 @@ external_select_name()
   fi
 }
 
+# Should we add an entry for truffleruby here? What's the purpose of this function?
```

Comment:
Should I add an entry for truffleruby here?
What's the purpose of this function?
Is it for guessing a good name based on the URL/folder given to `mount`?

`rvm mount -r 'https://github.com/oracle/truffleruby/releases/download/vm-1.0.0-rc2/truffleruby-1.0.0-rc2-linux-amd64.tar.gz'` seems to choose `truffleruby-1.0.0-rc2-linux-amd64` as the name currently.
It's mostly fine, except we might want to drop the `-linux-amd64`.
OTOH, it would then conflict with the normal name from `rvm install truffleruby`, which I'm sure is a good idea.
Maybe it should have some prefix or suffix when `rvm mount`-ed?

Reply:
see the comment in `scripts/functions/utility_system`